meta/main.yml

```
@@ -14,6 +14,7 @@ galaxy_info:
       versions:
         - "8"
         - "9"
+        - "10"
```

```suggestion
        - "10"
```
Ansible lint does not recognize "10" as a valid version.  In fact, Ansible is deprecating this version mechanism altogether.  `galaxy_tags` is now used for versions going forward.

---

Ah, ok. Good to know. I dropped the commit changing this and kept only the one fixing the repositories.